test/CollectionTest.cpp

```diff
@@ -0,0 +1,259 @@
+//
+// CollectionTest.cc
+//
+// Copyright © 2021 Couchbase. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "CBLTest.hh"
+#include "CBLPrivate.h"
+#include "fleece/Fleece.hh"
+#include "fleece/Mutable.hh"
+
+using namespace fleece;
+using namespace std;
+
+class CollectionTest : public CBLTest {
+    
+public:
+    
+    void createNumberedDocs(CBLCollection *col, unsigned n, unsigned start = 1) {
+        for (unsigned i = 0; i < n; i++) {
+            char docID[20];
+            sprintf(docID, "doc-%03u", start + i);
+            auto doc = CBLDocument_CreateWithID(slice(docID));
+            
+            MutableDict props = CBLDocument_MutableProperties(doc);
+            char content[100];
+            sprintf(content, "This is the document #%03u.", start + i);
+            FLSlot_SetString(FLMutableDict_Set(props, "content"_sl), slice(content));
+            
+            CBLError error;
+            bool saved = CBLCollection_SaveDocument(col, doc, &error);
+            CBLDocument_Release(doc);
+            REQUIRE(saved);
+        }
+    }
+    
+};
+
+TEST_CASE_METHOD(CollectionTest, "Default Scope and Collection Name Constant", "[Collection]") {
+    CHECK(kCBLDefaultCollectionName == "_default"_sl);
+    CHECK(kCBLDefaultScopeName == "_default"_sl);
```

Comment:
I don't think this needs to be a test.  How about a static assert instead?

Reply:
I guess a static assert won't work for this type, but it seems weird to have this as a runtime test.